packages/solarwinds-apm/src/shared/config.ts

```diff
@@ -208,7 +207,7 @@ export const schema = (defaults: Defaults) =>
       const otlp: Configuration["otlp"] = Object.fromEntries(
         (["traces", "metrics", "logs"] as const).map((signal) => [
           signal,
```

Comment:
kind of a tangent: on setting of headers (line 204), had a long [internal discussion](https://swicloud.atlassian.net/wiki/spaces/NIT/pages/3389030543/Custom+Configuration?focusedCommentId=5349245192) where i'm advocating that we only set the bearer auth header if there's no corresponding upstream header config AND the exporter endpoint is SWO.

Reply:
will address in a separate PR since it's not really related to SDK 2.0